src/components/Square.js

```diff
@@ -10,4 +11,13 @@ const Square = ({ id, className, space, name, stamp, flagPath }) => {
     )
 }
 
-export default Square;
\ No newline at end of file
+export default Square;
+
+Square.propTypes = {
```

Comment:
Do all these props, except for maybe flagpath, need to be required? I think if any were missing, functionality would break.

Reply:
We're not passing them in when the squares are empty, so they are not required